Sources/Fluent/Schema/Schema+Creator.swift

```diff
@@ -15,13 +15,14 @@ extension Schema {
         
         public func id(
```

Comment:
There needs to be some way to only have to specify id type once.

```swift
class User: Model {
    static var idType = .uuid
    ...

    static func prepare() {
        ...
        users.id(idType: .uuid)
        ...
    }
}
```

Having to repeat `.uuid` in two places isn't ideal.